src/Client/AkeneoPimClientBuilder.php

```diff
@@ -63,8 +63,11 @@ class AkeneoPimClientBuilder
      */
     public function __construct($baseUri, $clientId, $secret, $username, $password, $options = [])
     {
+        $accessToken = isset($options['access_token']) ? $options['access_token'] : null;
```

Comment:
You offer the possibility to set one parameter but not the other. It could lead to some problem if you only st the access_token for example.